Source/WebCore/platform/graphics/mac/controls/SwitchThumbMac.mm

```diff
@@ -89,15 +96,17 @@
 
     auto cgContext = trackBuffer->context().platformContext();
 
-    GraphicsContextStateSaver stateSaver(context);
```

Comment:
I don't think we should be removing this, there's still `context.drawConsumingImageBuffer` below. No guarantee that doesn't modify context state.

Reply:
I'm not sure what you mean. I moved it up so the scale change ends up being in scope.

Reply:
Sorry, I misread the diff.